server/bin/www

```diff
@@ -97,21 +97,34 @@ function onError(error) {
  */
 
 function onListening() {
+    logger.info("Setting up server event listener");
     const addr = server.address();
     const bind = typeof addr === "string" ? "pipe " + addr : "port " + addr.port;
-    console.log("Boop server listening on " + bind);
+    logger.info("Boop server listening on " + bind);
 }
 
 /**
  * Gets the configured login services
  */
-function getLoginService () {
+async function getLoginService () {
```

Comment:
Is async necessary for this function?

Reply:
No. This is a remnant of testing different methodologies while trying to ensure the order of the messages. Will remove it.